Classes/Domain/Repository/EntityNotificationRepository.php

```diff
@@ -50,4 +52,36 @@ public function findFromEventDefinition(EventDefinition $eventDefinition)
 
         return $query->execute();
     }
+
+    /**
+     * Returns the wanted notification even if it was disabled.
+     *
+     * @param mixed $identifier
+     * @return EntityNotification|object
+     */
+    public function findByIdentifierForce($identifier)
+    {
+        $query = $this->createQueryWithoutEnableStatement();
+
+        $query->matching(
+            $query->logicalAnd(
+                $query->equals('uid', $identifier),
+                $query->equals('deleted', false)
+            )
+        );
```

Comment:
Won't this only return the deleted rows?

Reply:
On the contrary, only the row when it has not been deleted in TYPO3. On the other hand, a disabled (hidden) row will be returned.